OpenOversight/tests/test_utils.py

```diff
@@ -19,13 +22,24 @@
 
 
 # Utils tests
-
 upload_s3_patch = patch(
     "OpenOversight.app.utils.cloud.upload_obj_to_s3",
     MagicMock(return_value="https://s3-some-bucket/someaddress.jpg"),
 )
 
 
+class PoliceDepartment:
+    """Base Police Department class."""
+
+    def __init__(self, name, short_name, state=random.choice(us.STATES).abbr):
```

Comment:
This will assign the same state to all departments created with this constructor which might not be intended?
Context: https://stackoverflow.com/questions/1132941/least-astonishment-and-the-mutable-default-argument

Reply:
Gooooood call! I can make that change.